_posts/0000-01-02-baelzbub.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Git and Github. For the Win!  
```

Comment:
Nice one :+1:  
But the guided lesson needs to see an edit before we can move on.